index.js

```diff
@@ -1,60 +1,80 @@
 'use strict';
 
-var https = require('https');
-var http = require('http');
+var request = require('request');
 
-https.globalAgent.maxSockets = 500;
+var defaultOptions = {
+  poll: {
+    maxSockets: 500,
+  },
+  timeout: 5000
+};
+
+// https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill
+if (typeof Object.assign != 'function') {
```

Comment:
Which nodejs version are you using? I'm thinking about making node-isbn compatible with nodejs 4+, that already comes with `Object.assign`. Sounds good?